bin/node-template/README.md

```diff
@@ -1,54 +1,76 @@
+[![Substrate Playground](https://img.shields.io/badge/Playground-node_template-brightgreen?logo=Parity%20Substrate)](https://playground-staging.substrate.dev/?deploy=node-template)
+
 # Substrate Node Template
 
-A new FRAME-based Substrate node, ready for hacking.
+A new FRAME-based Substrate node, ready for hacking :rocket:
 
-## Build
+## Local Development
 
-Install Rust:
+Follow these steps to prepare a local Substrate development environment :hammer_and_wrench:
 
-```bash
-curl https://sh.rustup.rs -sSf | sh
-```
+### Simple Setup
 
-Initialize your Wasm Build environment:
+Install all the required dependencies with a single command (be patient, this can take up
+to 30 minutes).
 
 ```bash
-./scripts/init.sh
+curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 
-Build Wasm and native code:
+### Manual Setup
+
+Find manual setup instructions at the [Substrate Developer Hub](https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-installation).
+
+### Build
+
+Once the development environment is setup, build the node template. This command will build the
```

Comment:
```suggestion
Once the development environment is set up, build the node template. This command will build the
```

Reply:
Should we change this on line 20 too then?

Reply:
I know I see